### PR TITLE
Create helm charts server

### PR DESCRIPTION
**Pull request by gianrubio**
@brancz could you create a s3 bucket to host all the charts? Please run the script bellow 

```
BUCKET_NAME=coreos-charts
aws s3api  create-bucket --bucket $BUCKET_NAME --create-bucket-configuration LocationConstraint=eu-west-1 --grant-read uri=http://acs.amazonaws.com/groups/global/AllUsers
CAT > website.json <<EOF
{
    "IndexDocument": {
        "Suffix": "index.html"
    },
    "ErrorDocument": {
        "Key": "error.html"
    }
}
EOF
aws s3api put-bucket-website --bucket $BUCKET_NAME --website-configuration file://website.json
```

After that run the Makefile command `make helm-init-repo && make helm-package` on this branch. 

#660

### CODE REVIEW

**Comment by brancz** [comment]
@ant31 could you give the helm charts in this repository a look and see if we can use the helm registry plugin for it? Even with extra steps would be fine, I would just prefer not to manually manage an s3 bucket.

**Comment by gianrubio** [comment]
friendly ping @ant31

File: helm/kube-prometheus/requirements.yaml [hunk 1]
```diff
@@ -1,44 +1,44 @@
...
 
   - name: exporter-kube-state
     version: 0.1.2
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
 
   - name: exporter-kubelets
     version: 0.1.0
-    repository: https://charts.cloudposse.com/incubator
+    repository: https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
```
Comment (ant31) [inline helm/kube-prometheus/requirements.yaml]: I don't see tthe export* in the index

Comment (gianrubio) [reply helm/kube-prometheus/requirements.yaml]: I don't get it, what you mean with exporter-*?

Comment (ant31) [reply helm/kube-prometheus/requirements.yaml]: export-*:  
```
exporter-kubelets
exporter-kube-state
exporter-kube-scheduler
exporter-kube-etcd
...
```
You point them the the s3 bucket.
In the s3 bucket there's the `index.yaml` ? 
in the `index.yaml` generated via the makefile from the PR, those subcharts are not indexed.

Comment (gianrubio) [reply helm/kube-prometheus/requirements.yaml]: I moved all the exporter to the root folder, also I added the `make helm-init-repo` to initialize the repo.
Plase run: `make helm-init-repo && make helm-package`

**Comment by brancz** [comment]
Sorry for the slow movement on this, we should soon get to this!

**Comment by gianrubio** [comment]
@brancz do you mind if I host the charts on my s3 bucket ?

**Comment by brancz** [comment]
I'm perfectly happy with that :slightly_smiling_face: .

**Comment by brancz** [comment]
Sorry about the inconvenience.

**Comment by ant31** [comment]
I've rebased and added the `dep update`:  #776 

Charts are pushed in:
https://s3-eu-west-1.amazonaws.com/coreos-charts/stable

**Pull request closed without merging**